cloudformation/template.yml

```diff
@@ -0,0 +1,339 @@
+      Environment:
+        default: Environment
+      Region:
+        default: AWS Region
+      AMI:
+        default: AMI
+      KeyName:
+        default: SSH Key Name
+      SpotFleetBidPercentage:
+        default: Spot Fleet Bid Percentage
+      MinimumVCPUs:
+        default: Minimum vCPU Count
+      DesiredVCPUs:
+        default: Desired vCPU Count
+      MaximumVCPUs:
+        default: Maximum vCPU Count
+      InstanceTypes:
+        default: Instance Types
+      JobDefinitionName:
+        default: Job Definition Name
+      InstanceVCPUs:
+        default: vCPU Limit
+      InstanceMemory:
+        default: Memory Limit
+      BatchServiceRolePolicyARN:
+        default: Batch Service Role
+      SpotFleetServiceRolePolicyARN:
+        default: Spot Fleet Service Role
+      RepositoryName:
+        default: Repository Name
+      ImageName:
+        default: Image Name
+      ImageTag:
+        default: Image Tag
+      VPC:
+        default: VPC
+      SubnetIds:
+        default: Subnets
+
+Parameters:
+  Project:
+    Type: String
+    Default: Raster Vision CloudFormation
+    Description: Name of the project associated with the Batch job
+
+  Environment:
+    Type: String
+    Default: Testing
+    Description: Environment of the project being deployed (e.g. Staging, Development, Production)
+
+  Region:
+    Type: String
+    Default: 'us-east-1'
+    Description: AWS Region in which to deploy the Batch job
+
+  AMI:
+    Type: String
+    Default: 'ami-06681f03ed7f21643'
+    Description: Amazon Machine Image to use for the compute environment
+
+  KeyName:
+    Type: 'AWS::EC2::KeyPair::KeyName'
+    Description: An Amazon EC2 key pair name to use for SSH access to the cluster
+
+  SpotFleetBidPercentage:
+    Type: Number
+    Default: 100
+    Description: >
+      Minimum percentage that a Spot Instance price must be when compared with
+      the On-Demand price for that instance type before instances are launched
+
+  MinimumVCPUs:
+    Type: Number
+    Default: 0
+    Description: The minimum number of EC2 vCPUs that an environment should maintain
+
+  DesiredVCPUs:
+    Type: Number
+    Default: 0
+    Description: The desired number of EC2 vCPUS in the compute environment
+
+  MaximumVCPUs:
+    Type: Number
+    Default: 80
+    Description: The maximum number of EC2 vCPUs that an environment can reach
+
+  InstanceTypes:
+    Type: List<String>
+    Default: p2.xlarge
+    Description: A comma-separated list of instance types that may be launched
+
+  JobDefinitionName:
+    Type: String
+    Default: 'raster-vision-gpu-testing'
+    Description: Specifies the name of the Batch Job Definition to create
+
+  InstanceVCPUs:
+    Type: Number
+    Default: 8
+    Description: Number of vCPUs reserved for the container by the task definition
+
+  InstanceMemory:
+    Type: Number
+    Default: 55000
+    Description: The hard limit (in MB) of memory to present to the container
+
+  BatchServiceRolePolicyARN:
```

Comment:
Given the intended audience, I could see there being value in omitting these policy ARNs from the parameters and just hardcoding them into the roles we create below.